lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3146,7 +3146,7 @@ def test_defaults(self, mock_set_custom_metric):
                 format='Homework',
             )
             vertical = ItemFactory.create(category='vertical', parent_location=subsection.location)
-            ItemFactory.create(category='problem', parent_location=vertical.location)
+            ItemFactory.create(category='problem', parent_location=vertical.location, has_score=True)
```

Comment:
This was the only spot that needed to be fixed? That feels like a small amount...